iotdb-cli/src/main/java/org/apache/iotdb/cli/client/WinClient.java

```diff
@@ -41,7 +41,6 @@
    */
   public static void main(String[] args) throws ClassNotFoundException, SQLException {
```

Comment:
This method seems quite similar to that in Client, may it can be extracted as a base method?

Reply:
In my opinion, we don't recommend reconstruct Client/WinClient in this PR:
1. This PR focus on addressing the sonar prompts to improve the code quality in code line level, but not package/class level.
2. It's right that `Client` is similar to and shorter than `WinClient`, nevertheless, this reconstruction might be more difficult than the other changes in the same PR. I recommend to fix it in next PR. @jt2594838 @MyXOF

Reply:
Surely you can put it afterwards, but it is still better to leave a "TODO" as a reminder.